### PR TITLE
Restructure draft-kuhn-quic-bdpframe-extension.xml

### PR DESCRIPTION
**Pull request by gorryfair**
partial Edit do not merge until asked to

### CODE REVIEW

**Comment by gorryfair** [comment]
Raffaello Secchi provided some feedback - which I all incorporate and then ask to merge:-)

**Comment by gorryfair** [comment]
Ready for review/merge.

File: draft-kuhn-quic-bdpframe-extension.xml [hunk 1]
```diff
@@ -314,82 +310,49 @@ the CC parameters -->
...
-        as a connection error of type TRANSPORT_PARAMETER_ERROR
-        <xref target="RFC9000"></xref>.</t>
+    <t> {XXX Editor Note: This presents the current transport
+    method to signal use and to send CC params. There are various alternatives
+    and the final choice is to be confirmed.}</t>
+
+   <t>The following subsections define four signals to carry transport control information.
+    The transmission of these signals is not limited by flow control limits
+    of the underlying QUIC transport. If a sender or receiver does not implement this
+   specification, the signals MSUT be ignored.</t>
```
Comment (NicoKos) [inline draft-kuhn-quic-bdpframe-extension.xml:322]: s/MSUT/MUST

File: draft-kuhn-quic-bdpframe-extension.xml [hunk 2]
```diff
@@ -430,87 +385,169 @@ CC_params {
...
+
+
+</section> <!--- CC Params -->
+	
+<section anchor="sec:avoid" title="Request to avoid using the saved CC Params (careful_resume_avoid)">
     <t>
-    A method is defined to exchange the CC params between
-    a QUIC receiver and the sender in an inter-operable manner. The process
-    is outlined here:
+    This ection describes a signal to the sender that requests to avoid using the saved cc params.
```
Comment (NicoKos) [inline draft-kuhn-quic-bdpframe-extension.xml:457]: s/ection/section

Comment (NicoKos) [reply draft-kuhn-quic-bdpframe-extension.xml:457]: s/cc params/CC parameters/

File: draft-kuhn-quic-bdpframe-extension.xml [hunk 3]
```diff
@@ -430,87 +385,169 @@ CC_params {
...
+</artwork>
+</figure>
+
     <list style="numbers">
-        <t>For an established connection, the current RTT, measured capacity,
+       	<t>When a receiver sends a enable_careful_resume_indication, a sender
+		in the Observe Phase of the Careful Resume method
+	is permitted to start sending CC params in a careful_resume_indication,
+	transported to the receiver using the QUIC connection.
+	Each inidcation includes the current RTT, measured capacity,
```
Comment (NicoKos) [inline draft-kuhn-quic-bdpframe-extension.xml:497]: s/inidcation/indication

**Comment by NicoKos** [comment]
The PR is great and the sequence helps understand the idea. 
I have just minor edition suggestions.

**Review by NicoKos — APPROVED** [review]

**Pull request merged**